File: 026. Remove Duplicates from Sorted Array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0:
            return 0
        flag = nums[0]
        index = 1
        for i in range(len(nums)):
            if nums[i] != flag:
                nums[index] = nums[i]
                flag = nums[i]
                index += 1
        return index 
                

File: 026. Remove Duplicates from Sorted Array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() == 0){
            return 0;
        }
        int flag = nums[0];
        int index = 1;
        for (int i = 1 ; i < nums.size() ; i++){
            if( flag != nums[i]){
                flag = nums[i];
                nums[index] = nums[i];
                index ++;
            }
        }
        return index;
    }
};